website/docusaurus.config.js

```diff
@@ -188,10 +188,20 @@ const siteConfig = {
     twitterImage: 'img/undraw_tweetstorm.svg',
 
     prism: {
-      additionalLanguages: ['ruby', 'csharp']
+      additionalLanguages: [
+        'ruby',
+        'rb',
```

Comment:
hey bud, just update `rb` to `rust` here and this change will be good to go.

Reply:
@YOU54F Oops, so sorry about that! Complete typo and I overlooked it initially. Meant for that to be rust, not rb, since ruby is already there. Sorry about that. Fixed it